TerminalDocs/TOC.yml

```diff
@@ -2,16 +2,48 @@
   items: 
     - name: Overview
       href: index.md
-    - name: Install
-      href: install.md
+    - name: Get started
+      href: get-started.md
     - name: Customize settings
       items:
```

Comment:
Placing this comment here so that it doesn't get lost:
Could you also add the following pages or topics somewhere:
- [ ] Selection
  - consolidate into one page to talk about selection when using the _mouse_.
  - maybe mention some info on keybindings too? And HTML copy?
  - copyOnSelect
- [ ] VT Mouse Mode
  - mention that we only support certain kinds of applications here
  - also that Shift lets you select, and just talk about the whole experience
- [ ] In troubleshooting:
  - https://github.com/microsoft/terminal/issues/3788#issuecomment-603539202

Reply:
There's a selection settings section on the global settings page which covers the selection bullets you've listed. We can add VTMM to the troubleshooting page.